templates/includes/transactions.tx

```diff
@@ -57,6 +57,13 @@
                     <div class="column-third column-status" id="status-<% $~transaction.count %>">
                         Status<strong> Processing </strong>
                     </div>
+                % } else if $transaction.status == "closed pending payment" {
+                    <div class="column-third column-status" id="status-<% $~transaction.count %>">
+                        Status<strong> Pending payment </strong>
+                        % if $transaction.resume_link && !$c.can_do('/admin/transaction-lookup') {
+                            <a class="piwik-event" data-event-id="resume" id="transaction-<% $~transaction.count %>-resume-link" href="<% $transaction.resume_link %>">PAY NOW</a>
```

Comment:
Can we make the data-event-id distinct? Something like 'pay to close transaction'? Would be helpful for analytics. Also the link should read 'Pay now' to keep the case consistent with the rest of the page